packages/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import type { CommonProps } from '@contentful/f36-core';
+
+import { getTabsStyles } from './Tabs.styles';
+export interface TabsProps extends CommonProps {
+  variant?: 'default' | 'horizontal' | 'vertical';
```

Comment:
I am a bit confused by these property values. Maybe it's just that it's about the divider and not really about the component itself and that's why I am confused. I think variant horizontal and vertical somehow suggest to me that it's about the whole component being vertical or horizontal. Dunno, maybe it's just me 🙃

Reply:
would it be better if property values would be called "verticalDivider" for example?

Reply:
It gives me clarity about what this variant actually does for the component, for sure. WDYT? Maybe other folks have a different perspective here. 🙂

Reply:
yeah, I'd probably go with `vertical-divider`, `horizontal-divider` and `default`